ffi/wgpu.h

```diff
@@ -118,14 +118,34 @@ typedef enum WGPUNativeQueryType {
     WGPUNativeQueryType_Force32 = 0x7FFFFFFF
 } WGPUNativeQueryType WGPU_ENUM_ATTRIBUTE;
 
+typedef enum WGPUDirectXMaxShaderModel {
```

Comment:
I'm not sure we should deviate from `wgpu`'s name here, without good reason - this is specifically about the max shader model of the DXC exe the user is handing to wgpu. I think this should either be `WGPUDx12ShaderModel` or `WGPUDxcMaxShaderModel`

Reply:
fixed